ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template4.kt

```diff
@@ -131,6 +134,13 @@ internal fun Template4(
                     }
                     .background(state.currentColors.background),
             ) {
+                if (landscapeLayout) {
```

Comment:
Hmm this is strange... So we only need to apply the margin in landscape?

Reply:
Yes, on portrait, there's an image

![image](https://github.com/RevenueCat/purchases-android/assets/664544/fb83c11a-f557-48f9-ba8d-1967d92606d7)

<img width="746" alt="Screenshot 2024-02-14 at 15 33 57" src="https://github.com/RevenueCat/purchases-android/assets/664544/45d6a05b-60bb-463f-9720-de6dd7b92008">